src/core/main.go

```diff
@@ -321,3 +326,37 @@ func getDefaultScannerName() string {
 	}
 	return ""
 }
+
```

Comment:
I suggest create a new folder to save this function or other system job in the future and import here like `systemjob.Register(ctx)` to keep the main clean and avoid too many business packages imported to main.